### PR TITLE
Removes Dwarfism

### PR DESCRIPTION
**Pull request by Flaborized**
[input wanted] [removal]

## About the PR 

This PR removes dwarfs completely, from both a trait miners can start with and a thing genetics can get in syringe-form.  I opted to remove code snippets instead of commenting them out, I figure that if they ever come up, the history is here and easy to find. Or at least to point people to. Don't gotta make the code a museum. 

## Why's this needed? 

Dwarfs are visually broken to the point where 'improve, don't remove' doesn't apply, imo. Not a single jumpsuit or outer suit is fit to them, they *all* look broken, even the stuff miners can *spawn* wearing. We seem to actually care a bit about this game looking good, and I think the dwarfs are pretty far outside 'looks good'.

There are of course, alternatives to removing them that don't make them look *as* broken. Custom fitting all clothing items to them (like monkeys) is completely unfeasible, it's too much work that'll never get done, even if it does, it'll make contributing clothing items more difficult, since you'd have to do two (three if you count monkey!! in the future when they're updated) sprites, which I don't think is worth it. Or that this solution would ever actually realistically be done or maintained.

The other (actually doable) alternative is using some kinda coding solution, like shrinking people's entire sprites down, like stims does or how obesity did before we cut it. While I agree this would certainly be a large improvement over what we have now, I think this would still look really bad and would clash really hard with all the pretty spriteart we've worked so hard on, enough so that I'd still rather have them completely gone than have some code-solution. 

I'm aware that some people enjoy dwarfs and don't mind them looking... broken. I don't think having dwarfs is a particularly bad idea at face value, but I *do* mind the broken art, and I think it's just not worth the aesthetic inconsistency to fix. We should have higher standards than that overall for How Things Look, or at least I see it that way. This is very much a specific Design Perspective though.

## Changelog

```changelog
(u)Flaborized
(*)Removed dwarfism.
```

### CODE REVIEW

**Comment by Camryn-Buttes** [comment]
I would like to request that the glowstick farting chance be moved to training_miner rather than outright removed.

**Comment by Gannets** [comment]
Agree with removal, it's a funny gag but never functioned in gameplay.

File: code/obj/item/clothing/hats.dm [hunk 1]
```diff
@@ -785,9 +785,6 @@
 
 	equipped(var/mob/user, var/slot)
 		..()
```
Comment (pali6) [inline code/obj/item/clothing/hats.dm:787]:
```suggestion
```
This can be removed entirely now.

**Comment by Flaborized** [comment]
> I would like to request that the glowstick farting chance be moved to training_miner rather than outright removed.

Done.

**Review by flrsh — APPROVED** [review]

**Comment by Kyle2143** [comment]
I just saw someone mention on discord that a solid way to keep dwarfs might be to instead change their sprite to have monkey proportions so that they can wear those clothes as well.

**Comment by Flaborized** [comment]
> I just saw someone mention on discord that a solid way to keep dwarfs might be to instead change their sprite to have monkey proportions so that they can wear those clothes as well.

**Comment by DimWhat** [comment]
> > I just saw someone mention on discord that a solid way to keep dwarfs might be to instead change their sprite to have monkey proportions so that they can wear those clothes as well.
> 
> 

Finally, you can now play as a baby.

**Review by mordent-goonstation — APPROVED** [review]

**Pull request merged**